src/persistence.ts

```diff
@@ -15,13 +15,15 @@ import {
  * Persistence layer to save tapes to disk and read them from disk.
  */
 export class Persistence {
-  constructor(private tapeDir: string) {}
+  constructor(private tapeDir: string, private redactHeaders: string[]) {}
```

Comment:
Suggestion: These two could probably be readonly, might as well :)
```suggestion
  constructor(private readonly tapeDir: string, private readonly redactHeaders: string[]) {}
```